src/main/pom-parser/pom-parser.js

```diff
@@ -1,9 +1,9 @@
 const xml2js = require('xml2js');
 
 const getCoordinatesFromNode = (node) => {
-  const groupId = node.groupId ? node.groupId[0] : '';
-  const artifactId = node.artifactId ? node.artifactId[0] : '';
-  const version = node.version ? node.version[0] : '';
+  const groupId = node.groupId ? node.groupId[0] : (node.parent ? getCoordinatesFromNode(node.parent[0]).groupId : undefined );
```

Comment:
Actually, based on https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Inheritance , it looks like this works for the version as well, so let's check and make sure that we inherit version as well.

There's an interesting caveat to this that because it only works in situations where there is a parent at the same level meaning that parent elements and dependency elements are safe.  We might want to add a comment here to indicate that for parent or dependencies, the parent check won't matter because it's not possible.